account_invoice_delivered_amount/models/account_invoice.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    total_delivered_amount = fields.Monetary(
```

Comment:
```suggestion
    amount_total_delivered = fields.Monetary(
```
Following the convention of other amount_xxx fields.